challengers/Joyal-Poudel/day4-cybersecurity-password-strength/index.html

```
@@ -1,3 +1,4 @@
+pull request
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)